includes/class-wc-stripe-payment-method-configurations.php

```diff
@@ -143,6 +143,27 @@ public static function get_parent_configuration_id() {
 		return self::get_primary_configuration()->parent ?? null;
 	}
 
+	/**
+	 * Get the UPE available payment method IDs.
+	 *
+	 * @param bool $force_refresh Whether to force a refresh of the payment method configuration from Stripe.
+	 * @return array
+	 */
+	public static function get_upe_available_payment_methods( $force_refresh = false ) {
```

Comment:
This is similar to `get_upe_enabled_payment_method_ids` but for all available methods, I think we should keep the function names consistent by renaming this to `get_upe_available_payment_method_ids`, we are actually returning just the ids not the full payment methods